packages/cli/src/lib/cmds/build.ts

```diff
@@ -137,5 +139,31 @@ export async function build(
   );
 
   log.info(`Signed Android App generated at "${outputFile}"`);
+
+  try {
+    const digitalAssetLinksFile = './assetlinks.json';
+    const keyInfo = await keyTool.keyInfo({
+      path: twaManifest.signingKey.path,
+      alias: twaManifest.signingKey.alias,
+      keypassword: passwords.keyPassword,
+      password: passwords.keystorePassword,
+    });
+
+    const sha256Fingerprint = keyInfo.fingerprints.get('SHA256');
+    if (!sha256Fingerprint) {
+      log.warn('Could not find SHA256 fingerprint. Skipping generating "assetlinks.json"');
+    } else {
+      const digitalAssetLinks =
+      DigitalAssetLinks.generateAssetLinks(twaManifest.packageId, sha256Fingerprint);
+
+      await fs.promises.writeFile(digitalAssetLinksFile, digitalAssetLinks);
+
+      log.info(`Digital Asset Links file generated at ${digitalAssetLinksFile}`);
+      log.warn('If opting into "App Signing by Google Play" when uploading the Android app for ' +
```

Comment:
I understand why you're doing this, but I don't like the idea of the tool outputting a warning that the user can't get rid of. I'm not sure there's a better way though.

Reply:
We can do info and just hightlight it with colors. How does that sound?

Reply:
I think it's more that even if the user does everything right, they're going to have a big highlighted warning every time they run the tool.

I think longer term we should have something in twa-config for Play Signing - either a "isUsingPlaySigning: false" the user can set or a "playSigningSignatures: '...'" they can use to make things work right. It's probably outside of the scope for this change though.